src/main/java/org/rumbledb/expressions/control/SwitchExpression.java

```diff
@@ -81,16 +80,14 @@ public String serializationString(boolean prefix) {
         StringBuilder result = new StringBuilder();
         result.append("(SwitchExpression switch ");
         result.append(this.testCondition.serializationString(false));
-        for(SwitchCase c : this.cases)
-        {
-        	for(Expression e : c.getConditionExpressions())
-        	{
-        		result.append("case ");
-        		result.append(e.serializationString(false));
-        		result.append(" ");
-        	}
-    		result.append("return ");
-    		result.append(c.getReturnExpression().serializationString(false));
+        for (SwitchCase c : this.cases) {
+            for (Expression e : c.getConditionExpressions()) {
+                result.append("case ");
+                result.append(e.serializationString(false));
+                result.append(" ");
+            }
+            result.append("return ");
+            result.append(c.getReturnExpression().serializationString(false));
```

Comment:
This will potentially not cause issues with merging but just a pointer: The formatting was already fixed in the latest commit of SwitchFix. I believe this branch is not up to date with SwitchFix (target branch) and shows a larger diff.

Reply:
Yes, I was aware of this. This should be good now.